client/src/views/account/AccountAuthenticationPage.vue

```diff
@@ -0,0 +1,174 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div
+    v-if="ParsecAccount.isLoggedIn()"
+    class="account-authentication-page-container"
+  >
+    <div class="authentication-method-list">
+      <div
+        v-for="method of authMethods"
+        :key="method.authMethodId"
+        class="authentication-method-item"
+        :class="{ 'authentication-method-item--current': method.current }"
+      >
+        <div class="authentication-method-item-icon">
+          <ms-image
+            :image="method.usePassword ? EllipsisGradient : KeypadGradient"
+            class="authentication-method-item-icon__image"
+          />
+        </div>
+        <div class="authentication-method-item-details">
+          <ion-text class="authentication-method-item-details__title subtitles-normal">
+            {{ method.usePassword ? 'Password' : 'Other' }}
+          </ion-text>
+          <ion-text
+            class="authentication-method-item-details__date body"
+          >
+            <span>{{ $msTranslate('HomePage.profile.authentication.createdOn') }}</span>
+            <span>{{ $msTranslate(I18n.formatDate(method.createdOn, 'short')) }}</span>
+          </ion-text>
+        </div>
+        <ion-icon
+          v-if="method.current"
+          class="authentication-method-item-checkmark"
+          :icon="checkmarkCircle"
+        />
+      </div>
+    </div>
+
+    <ion-button
+      v-if="authMethods.find((method) => method.usePassword === true && method.current) !== undefined"
+      @click="updatePassword"
```

Comment:
What's the point of this button and the end goal? `Change method` is just `Change password` here. Is the goal to be able to switch between registered authentication methods, which would explain why they're displayed on the page? And eventually to change the password?